x/records/module_ibc.go

```diff
@@ -179,17 +179,18 @@ func (im IBCModule) OnAcknowledgementPacket(
 	// Custom ack logic only applies to ibc transfers initiated from the `stakeibc` module account
 	// NOTE: if the `stakeibc` module account IBC transfers tokens for some other reason in the future,
 	// this will need to be updated
+	err := im.keeper.ICACallbacksKeeper.CallRegisteredICACallback(ctx, packet, &ack)
+	if err != nil {
+		errMsg := fmt.Sprintf("Unable to call registered callback from records OnAcknowledgePacket | Sequence %d, from %s %s, to %s %s | Error %s",
+			packet.Sequence, packet.SourceChannel, packet.SourcePort, packet.DestinationChannel, packet.DestinationPort, err.Error())
+		im.keeper.Logger(ctx).Error(errMsg)
+		return sdkerrors.Wrapf(icacallbacktypes.ErrCallbackFailed, errMsg)
+	}
+
+	// log the ack type
 	switch resp := ack.Response.(type) {
```

Comment:
since all this does is log the ack type, might make sense to move this above `CallRegisteredICACallback`

Reply:
done